packages/docs/src/pages/guidelines/accessibility.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
+- include accessibility as part of our design process when working on new or existing components.
```

Comment:
maybe this bullet and the one below can be combined? both are talking about new components

Reply:
Maybe updating "include accessibility as part of our design process when working on new or existing components." to something like "include accessibility as part of our design process." would still give enough details and also allow us to keep both bullet points?

Reply:
Maybe... I feel like this bullet doesn't really say anything specific. And  honestly, accessibility should be included in design/product/engineering processes as well. The way USWDS handles this is by linking to [Accessibility for teams](https://accessibility.digital.gov/) which has some details and guidance for different roles.

Reply:
I like the idea of linking out to the accessibility for teams site - I'll include that!
Maybe we can remove the 2nd bullet point altogether then?